src/core/Akka.Tests/Actor/ActorSystemSpec.cs

```diff
@@ -238,6 +239,50 @@ public void Support_using_a_customer_scheduler()
             var actorSystem = ActorSystem.Create(Guid.NewGuid().ToString(), DefaultConfig.WithFallback("akka.scheduler.implementation = \"Akka.Tests.Actor.TestScheduler, Akka.Tests\""));
             Assert.True(actorSystem.Scheduler.GetType() == typeof(TestScheduler));
         }
+
+        [Fact]
+        public void Allow_configuration_of_guardian_supervisor_strategy()
+        {
+            var config = ConfigurationFactory.ParseString("akka.actor.guardian-supervisor-strategy=\"Akka.Actor.StoppingSupervisorStrategy\"")
+                .WithFallback(DefaultConfig);
+
+            var system = ActorSystem.Create(Guid.NewGuid().ToString(), config);
+
+            var a = system.ActorOf(actor =>
+            {
+                actor.Receive<string>((msg, context) => { throw new Exception("Boom"); });
+            });
+
+            var probe = CreateTestProbe(system);
+            probe.Watch(a);
+
+            a.Tell("die");
+
+            var t = probe.ExpectTerminated(a);
```

Comment:
Yeah, this looks fine to me - don't think we need #1753 to complete this. The `ExpectTerminated` does the job.